src/krux/pages/login.py

```diff
@@ -237,6 +234,8 @@ def _load_key_from_keypad(
                         autocomplete_fn,
                         possible_keys_fn,
                     )
+                    if word == "Esc":
```

Comment:
This should be `display.ESC` so translated strings will also match (if different)

Reply:
My idea was not involve translations on this, there's no t() method here, but I will define a ESC_CODE constant to in the code

Reply:
Just used MENU_CONTINUE constant instead, it is the command that Esc executes and won't conflict with any key, worked fine.